lib/services/Service.js

```diff
@@ -94,7 +94,7 @@ Service.prototype._request = function (action, variables, callback) {
  * @return {void}
  */
 Service.prototype._parseKey = function (input, key, callback) {
-  debug('Service._parseKey(%j, %j)', key, callback)
+  debug('Service._parseKey(%j, %j, %j)', input, key, callback)
```

Comment:
This line makes it very verbose. Can you put it in as a comment, for others to use?

Reply:
Sure